### ISSUE TITLE
Design embedded search for administrative fines on enforcement landing page

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#4823]
**What we're after:**
Design an embedded search for the [enforcement landing page](https://www.fec.gov/legal-resources/enforcement/). This will give users an opportunity to do a quick search, similar to the [MURs on that page](https://www.fec.gov/legal-resources/enforcement/#programs). 

**Design considerations:**
- High level search options: number and keyword
- Text to link to the EQS search (since that still has functionality we need)
- Incorporate new link to the calculator?

**Completion criteria:**
- [x] New embedded search for administrative fines is designed and reviewed
- [ ] Followup implementation ticket is created

**Comment by JonellaCulmer** [comment]
Draft embedded search for enforcement page: 
![Screen Shot 2021-08-06 at 10 51 19 AM](https://user-images.githubusercontent.com/31663028/128529329-1d6b153b-09f5-41ad-b611-eed61b3649d8.png)

**Comment by JonellaCulmer** [comment]
@patphongs @dorothyyeager @kathycarothers In the previous comment, you'll find a mockup of the imbedded administrative fine search. Please let me know if you have any comments/concerns/adjustments. I modeled the language at the bottom of the feature to "match" what we used for the [embedded MUR search](https://www.fec.gov/legal-resources/enforcement/).

**Comment by kathycarothers** [comment]
@JonellaCulmer it looks good. I like the added function to search for cases. I don't have a problem with the mock up.

**Comment by JonellaCulmer** [comment]
@kathycarothers FYI I updated the mockup to say 2000 instead of 2001.

**Comment by dorothyyeager** [comment]
@JonellaCulmer Looks great. I'd suggest we say "case number" instead of "administrative fine  number." The case numbers usually start with AF, so would suggest updating the example to say AF1234 (the case numbers usually have four digits). Here's a screenshot from the current search.

![image](https://user-images.githubusercontent.com/24437369/130621189-819cb1c1-3c4b-4cb9-9241-fbc22098cdbc.png)

**Comment by patphongs** [comment]
@JonellaCulmer I agree with all of @dorothyyeager comments and I think this is technically feasible to do. Thanks!

**Comment by JonellaCulmer** [comment]
@dorothyyeager @patphongs Thanks for those comments. I updated the mockup. Unfortunately, there is no case 1234, so I changed the example to 2345 instead. We tend to include actual cases, keywords, etc so that folks can test and know what to expect with the results. 

<img width="984" alt="Screen Shot 2021-08-24 at 9 40 44 AM" src="https://user-images.githubusercontent.com/31663028/130627601-72f440bf-fa63-466c-9124-fe1931831cb8.png">
